CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+## Table of Contents
+1. [About the Project](#about-the-project)
+1. [Questions and Help](#questions-and-help)
+1. [Getting Started](#getting-started)
+1. [Useful Resources](#useful-resources)
+
+# About the Project
+
+This is an Extension for [VSCode](https://code.visualstudio.com/) (Visual Studio Code), a cross-platform text editor, to support Bethesda's Papyrus scripting language. It is written primarily in TypeScript and C#.
+
+Write mods for Fallout 4 and Skyrim with the aid of modern code editing features such as code completion, jump to definition, hover, symbol search and live updated diagnostics.
+
+[![](https://vsmarketplacebadge.apphb.com/version-short/joelday.papyrus-lang-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=joelday.papyrus-lang-vscode)
+
+Uses the **[Papyrus Debug Adapter](https://github.com/joelday/papyrus-debug-server) xSE plugin** for live debugging.
+
+# Questions and Help
+
+Do you have questions or need help? Please come visit the....
+[Papyrus Language Tools Discord
+![Discord](https://img.shields.io/discord/558746231665328139.svg?color=%23738ADB)](https://discord.gg/E4dWujQ)
+
+# Getting Started
+
+## Building
+
+First, you will need Windows with the following installed on your system:
+- VSCode [Download and install](https://code.visualstudio.com/)
+- Node.js [Download and install](https://nodejs.org/)
+- Git for Windows [Download and install](https://git-scm.com/download/win)
+- .NET Core 3.0 [Download and install](https://dotnet.microsoft.com/download/dotnet-core)
```

Comment:
Don't think this is needed yet. Visual Studio 2018 or 2019 (for the debugger) is recommended.

Reply:
ok I'll remove Core 3.0 and add VS 2019. (Only reason I substituted the .net sdk for VS is I was trying to figure out the minimal stuff needed for build.ps1 to work.) What would be the minimal dependencies for someone who wants build to work but isn't going to touch the C#?